app/models/task.py

```diff
@@ -1,5 +1,41 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column, relationship
 from ..db import db
+from .goal import Goal
+from datetime import datetime
+from typing import Optional
 
 class Task(db.Model):
     id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
+    title: Mapped[str] = mapped_column(db.String, nullable=False)
+    description: Mapped[str] = mapped_column(db.String, nullable=False)
```

Comment:
_""...Columns that are part of the table are declared, by adding attributes that include a special typing annotation called [Mapped](https://docs.sqlalchemy.org/en/20/orm/internals.html#sqlalchemy.orm.Mapped). The name of each attribute corresponds to the column that is to be part of the database table. The datatype of each column is taken first from the Python datatype that’s associated with each [Mapped](https://docs.sqlalchemy.org/en/20/orm/internals.html#sqlalchemy.orm.Mapped) annotation; int for INTEGER, str for VARCHAR, etc. Nullability derives from whether or not the Optional[] type modifier is used."_ - [SQLAlchemy Documentation](https://docs.sqlalchemy.org/en/20/orm/quickstart.html#declare-models:~:text=Nullability%20derives%20from%20whether%20or%20not%20the%20Optional%5B%5D%20type%20modifier%20is%20used.)
```suggestion
    title: Mapped[str]
    description: Mapped[str]
```